keyboards/massdrop/ctrl/keymaps/r-pufky/README.md

```diff
@@ -0,0 +1,33 @@
+# Massdrop Ctrl r-pufky
```

Comment:
Please rename this file to `readme.md`.

You may need to rename it twice (e.g. `README.md` -> `read me.md` -> `readme.md`) to get it to stick.